src/multibody/geometry.hpp

```diff
@@ -220,20 +232,35 @@ namespace se3
     std::vector <DistanceResult> distance_results;
     
     ///
+    /// \brief Defines what information should be computed by collision test.
+    ///
+    fcl::CollisionRequest collisionRequest;
```

Comment:
Nope. It just seems very logical to do it. If you dont see any use, lets not do it :)